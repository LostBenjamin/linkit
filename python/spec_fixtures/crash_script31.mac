
[macro]
  (((waste 20)))

[[waste INSTANCE
  12345,12345,12345
  sto
]]

