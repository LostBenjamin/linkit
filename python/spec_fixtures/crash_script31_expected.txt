Macro being tested exceeds char buffer size

